packages/spectral-test/src/ComponentReference.test-d.ts

```diff
@@ -13,7 +13,7 @@ type AllComponentReferences =
   | ActionReference;
 
 const createRef = <TComponentRef extends AllComponentReferences>(
-  ref: TComponentRef
+  ref: TComponentRef,
```

Comment:
huge nit here: is this from biome formatter? just seems odd to have a , here

Reply:
Yup adds trailing commas